doc/permissions.md

```diff
@@ -3,7 +3,13 @@
 This document explains the need for each [extension permission](https://developer.chrome.com/extensions/declare_permissions) declared in Privacy Badger's [extension manifest](/src/manifest.json).
 
 ## Privacy
-The Privacy API lets extensions modify browser-wide privacy settings. Privacy Badger uses the Privacy API to disable [hyperlink auditing](https://www.bleepingcomputer.com/news/software/major-browsers-to-prevent-disabling-of-click-tracking-privacy-risk/). Privacy Badger also disables suggestions for similar pages when a page can't be found in Chrome, as this Chrome feature sends visited web addresses to Google. In addition, Privacy Badger allows users to set a stricter WebRTC IP handling policy in order to prevent leaking local network address information.
+The Privacy API lets extensions modify browser-wide privacy settings. Privacy Badger uses the Privacy API to disable the following settings:
+
+- [hyperlink auditing](https://www.bleepingcomputer.com/news/software/major-browsers-to-prevent-disabling-of-click-tracking-privacy-risk/)
```

Comment:
This isn't blocking for approval imo, but I do wonder if hyperlink auditing deserves a short blurb description like the other two points have